### PR TITLE
Added descriptions to all Contributor Experience subprojects

### PR DESCRIPTION
**Pull request by jberkus**
…are my own interpretation based on the files owned by each subproject, and as such need revision and editing.

Does not include generated file because I expect discussion before this PR is merged; this is more being used to start that discussion.

/hold
/sig contributor-experience
/priority important-soon
/kind docs

Please examine and revise:

@parispittman @Phillels @cblecker @nikhita

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -1062,12 +1062,15 @@ sigs:
       description: Test Failures and Triage
   subprojects:
   - name: community
+    description: Owns and manages overall community repo, including SIG description and charter maintenance.
```
Comment (parispittman) [inline sigs.yaml]: I would note this as 'owns and manages overall community repo, including community group documentation and operations.' I think SIGs own their own charter maintenance and descriptions at the end of the day.

File: sigs.yaml [hunk 2]
```diff
@@ -1062,12 +1062,15 @@ sigs:
...
     owners:
     - https://raw.githubusercontent.com/kubernetes/community/master/OWNERS
   - name: community-management
+    description: Manages all communications mechanisms for the community, including Slack, mailing lists, Zoom, social media, and YouTube.
```
Comment (parispittman) [inline sigs.yaml]: Can we be more broad here? We don't own social media like Twitter and SIGs technically own their own lists, we help moderate in certain cases, and own others like k-dev. 

'operations and policy for upstream community group communication platforms'

**Comment by jberkus** [comment]
All feedback has been incorporated.  Ready to go!

/remove-hold

**Comment by parispittman** [comment]
/lgtm

looks like you need to take my tag out of the last commit message; otherwise, this is ready to go. will wait for another to approve and the tag removal. thanks again - appreciate it.

**Comment by cblecker** [comment]
/approve

Gonna have tide squash this to get it in :)

**Pull request merged**